### PR TITLE
Changes brain damage lines

### PR DESCRIPTION
**Pull request by MrJWhit**
## About The Pull Request

Adds the noise emotes, as well as makes the brain damage not lean towards OOC in IC language.

## Why It's Good For The Game

The "ahelp" and "looc" lines always felt a bit weird to me, I'm fine with the misspelling and grammatical fuckups because it plays into everything. Also adds the audio brain damage, because, hey, it's fun.
I'm keeping "lifelike textures ;_;" because that's an amazing line, and I will go to my grave liking it.

## Changelog
:cl:
tweak: tweaked brain damage line
/:cl:

### CODE REVIEW

**Comment by GrayRachnid** [comment]
ooc dude rly

File: strings/traumas.json [hunk 1]
```diff
@@ -112,29 +108,29 @@
...
-		"ahelp help security powergaming with mindshields???",
 		";,g absorbing @pick(roles)",
 		";chemist can u @pick(create_verbs) holy @pick(mellens) for @pick(s_roles)???!!",
 		"@pick(semicolon) LIZZARRD SPEAKIGN IN EVIL BULL LANGUAGE SCI!!",
 		"@pick(semicolon)POST REBOOT MESSAGE LOLOL FUCK FUCK FUCK YOU",
 		"@pick(semicolon)so, i was trying to talk to someone on rp today, and then a mime walks up and pies them in the face along with some other prankster--i thought that mimes and clowns are supposed to be hired to entertain not to be a nuisance, and that if entertainment comes at someone elses expense then it's not supposed to be done. is that enough to like submit a player complaint or some shit or am i just being petty?"
+		"@pick(semicolon)*nya",
+		"@pick(semicolon)*awoo",
+		"@pick(semicolon)*merp"
+		"@pick(semicolon)*weh"
```
Comment (Ghommie) [inline strings/traumas.json]: This is abuse of brain-damaged mobs, I do not approve.

**Comment by IronEleven** [comment]
"ahelp hos pwrgamre they have 24 inch dick",

RIP

**Comment by Poojawa** [comment]
Why are you removing Citadel history

**Comment by MrJWhit** [comment]
It's not removing history, it's just making there not be ooc speak in ic.

**Comment by silicons** [comment]
i don't like this

**Comment by davidchan13** [comment]
Why isn't *dab on the brain damage list?

**Comment by MrJWhit** [comment]
> i don't like this

Got any, you know, a reason for this?


> Why isn't *dab on the brain damage list?

*dab previously had brain damage attached to it, and it hasn't been long enough that I feel comfortable tagging it. Someone could make a fix for people dabbing to death in cuffs, and then the brain damage portion is re-added. For something pretty minor like brain damage lines I'd rather give it more time if someone wants to mess with dabbing.

**Review by silicons — APPROVED** [review]

**Pull request merged**